frontends/web/src/routes/account/account.tsx

```diff
@@ -217,7 +208,7 @@ class Account extends Component<Props, State> {
     }
 
     private export = () => {
-        if (this.state.fatalError) {
+        if (this.state.status === undefined || this.state.status.fatalError) {
```

Comment:
fine for now, but in the future I would like to hide the export button or make it disabled,  if app has no status yet or has a fatalError